product_attribute_priority/views/product_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="attribute_tree_view">
+        <field name="model">product.attribute</field>
+        <field name="inherit_id" ref="product.attribute_tree_view"/>
+        <field name="arch" type="xml">
+            <field name="name" position="before">
+                <field name="sequence" widget="handle"/>
```

Comment:
Please remove this line, because odoo implements it, now
https://github.com/odoo/odoo/blob/9.0/addons/product/product_view.xml#L532